README.md

```diff
@@ -35,22 +38,124 @@ const wssOtions = {
 
 fastify.register(require('fastify-websocket'), {
   handle,
-  options: wssOptions
+  options: { maxPayload: 1048576 }
 })
 
-function handle (conn) {
+function handle(conn) {
```

Comment:
Standard style puts spaces after function declaration. I think it is better to fix where the space is missing instead of removing it.